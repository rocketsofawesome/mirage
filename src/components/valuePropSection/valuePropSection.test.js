import React from 'react'
import 'jest-styled-components'

import { ValuePropSection } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) ValuePropSection', () => {
  const createValuePropSection = (props) => {
    return shallowWithTheme(<ValuePropSection evergreenPromoItemCount='6' evergreenPromoPercent='15' {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createValuePropSection())
    .toMatchSnapshot()
  })
})
