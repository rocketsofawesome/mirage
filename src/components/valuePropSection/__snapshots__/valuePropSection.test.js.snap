// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) ValuePropSection matching the snapshot 1`] = `
.c0 .c1 {
  text-align: center;
  text-transform: uppercase;
}

.c0 .c1 {
  font-size: 24px;
  margin: 0 auto;
  margin-bottom: 40px;
}

.c0 .c2 {
  text-align: center;
  max-width: 275px;
  margin: 0 auto;
  font-size: 16px;
}

.c0 .image {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: auto;
  min-height: 76px;
  text-align: center;
}

.c0 .flexDiv {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

.c0 .innerFlexDiv {
  margin-bottom: 60px;
}

.c0 .innerFlexDiv:last-child {
  margin-bottom: 0;
}

.c0 .textTitle {
  font-weight: bold;
}

@media (max-width:958px) {
  .c0 .c1 {
    font-size: 3.4rem;
  }
}

@media (min-width:769px) {
  .c0 .c1 {
    font-size: 32px;
  }
}

@media (min-width:961px) {
  .c0 .innerFlexDiv:nth-last-child(-n + 3) {
    margin-bottom: 0;
  }
}

<BaseValuePropSection
  className="c0"
  constrained={true}
  padding={true}
/>
`;
